config.json

```diff
@@ -44,7 +44,8 @@
   ],
   "ignored": [
     "_test",
-    "docs"
+    "docs",
+    "img"
```

Comment:
note for us, https://github.com/exercism/configlet/pull/5/files made this explicit add unnecessary - do we still want it anyway?